docs/src/examples/components/Alert/Variations/AlertExampleOOF.shorthand.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { Alert } from '@stardust-ui/react'
+
+const AlertExampleInfo = () => (
+  <Alert
+    oof
```

Comment:
We don't have the` oof` as a prop in the component, so this will end up as a unhandled prop on the generated HTML dom element. This is not something that we want. I agree that these are Teams specific, so as a first step would recommend using boolean variable for this with the same name, instead of property.

Reply:
Not sure if these should be publicly visible as an examples though if we move the props to variables, as they are becoming theme specific. Anyway, we can add the examples and show them on the docs, and eve add steps for testing them in different themes. @Bugaa92 @kuzhelov can I have your input on this as well.

Reply:
I updated `oof` and `urgent` to use a variable instead

Reply:
agreed with @mnajdova , `oof` and `urgent` should be variables for now
@codepretty what happened with `critical` variation?